tkg/client/init.go

```diff
@@ -273,6 +273,15 @@ func (c *TkgClient) InitRegion(options *InitRegionOptions) error { //nolint:funl
 		targetClusterNamespace = options.Namespace
 	}
 
+	if config.IsFeatureActivated(constants.FeatureFlagPackageBasedCC) {
+		// for cc based cluster, check if tkr/cbt is reconciled to avoid package not found error by addons-controller
+		log.Info("Checking Tkr in bootstrap cluster...")
+		_, err = bootStrapClusterClient.GetTanzuKubernetesReleases("")
```

Comment:
Getting a list of TKrs might not be sufficient. We need to check for the presence of TKr that is going to be used for creation of management cluster.

There is a configuration`data.values.KUBERNETES_VERSION` . I am not sure how to get it in CLI but we need to be able to get it and then use that to get the TKr.

Reply:
thanks, lemme figure it out